src/Microsoft.MobileBlazorBindings/PageNavigationManager.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using XF = Xamarin.Forms;
+using MBB = Microsoft.MobileBlazorBindings;
+using Xamarin.Forms;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.AspNetCore.Components;
+
+namespace Microsoft.MobileBlazorBindings
+{
+
+    //This class was mainly for testing some concepts. It's not fully implemented but it's not far off.
```

Comment:
Do you think the state of this is useful enough to keep for now? Or is it better to table it and try again in the future?

Reply:
I think we throw it in the bin.